README.md

```diff
@@ -48,13 +48,16 @@ The following are all _required_.
 |---|---|---|
 |`environment`|Set the environment for this release. E.g. "production" or "staging". Omit to skip adding deploy to release.|-|
 |`finalize`|When false, omit marking the release as finalized and released.|`true`|
+|`ignore_missing`|When the flag is set and the previous release commit was not found in the repository, will create a release with the default commits count instead of failing the command.|`false`|
+|`ignore_empty`|When the flag is set, command will not fail and just exit silently if no new commits for a given release have been found.|`false`|
```

Comment:
Feels bad to duplicate this between the README and `action.yml`, but I realized that there are slight differences between the readme and this (for ex, `version` defaults to `{{github.sha}}`, but there is not default field in yml for that key).